subsys/bluetooth/mesh/Kconfig

```diff
@@ -1477,25 +1477,21 @@ config BT_MESH_USES_MBEDTLS_PSA
 	select EXPERIMENTAL
 	select MBEDTLS
 	select MBEDTLS_PSA_CRYPTO_C
-	select MBEDTLS_ENTROPY_C
 	select PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_IMPORT
 	select PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_EXPORT
 	select PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_GENERATE
 	select PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_DERIVE
 	select PSA_WANT_ALG_CMAC
 	select PSA_WANT_ALG_ECB_NO_PADDING
 	select PSA_WANT_KEY_TYPE_AES
-	select MBEDTLS_CIPHER_AES_ENABLED
 	select MBEDTLS_AES_ROM_TABLES
 	select PSA_WANT_ALG_CCM
 	select PSA_WANT_KEY_TYPE_HMAC
 	select PSA_WANT_ALG_HMAC
 	select PSA_WANT_ALG_SHA_256
 	select PSA_WANT_ALG_ECDH
 	select PSA_WANT_ALG_ECDSA
```

Comment:
I guess `PSA_WANT_ALG_ECDSA` is not required either. This is for signatures those Mesh doesn't use. Previously, mbedtls united it with ECDH.

Reply:
Removed it now